packages/espree/test-d/index.test-d.ts

```diff
@@ -0,0 +1,59 @@
+import { expectType } from 'tsd';
+
+import * as espree from "../espree.js";
+import type { Options, EspreeTokens } from "../espree.js";
+import type {VisitorKeys} from "../../eslint-visitor-keys/lib/index.js";
+
+const ast = espree.parse("let foo = \"bar\"");
+expectType<acorn.Node>(ast);
+
+const ast_option = espree.parse("let foo = \"bar\"", { ecmaVersion: 6 });
+expectType<acorn.Node>(ast_option);
+
+const tokens = espree.tokenize("let foo = \"bar\"");
+expectType<EspreeTokens>(tokens);
+
+const tokens_option = espree.tokenize("let foo = \"bar\"", { ecmaVersion: 6 });
+expectType<EspreeTokens>(tokens_option);
+
+const name = espree.name;
+expectType<string>(name);
```

Comment:
I think this should be either:

```suggestion
name satisfies string;
```

or:


```suggestion
expectType<"espree">(name);
```

And similarly for the other failing assertions below.